runtime/android/core_internal/src/org/xwalk/core/internal/XWalkSettingsInternal.java

```diff
@@ -1130,6 +1131,27 @@ private boolean getTextAutosizingEnabledLocked() {
         return mLayoutAlgorithm == LayoutAlgorithmInternal.TEXT_AUTOSIZING;
     }
 
+    /**
+     * Sets whether the XWalkView should support the wide viewport quirks mode.
+     * @param enable whether XWalkView should support wide viewport quirks mode.
+     * @since 6.0
+     */
+    @XWalkAPI
+    public void setWideViewportQuirkEnabled(boolean enable) {
```

Comment:
I have thought it. In ```src/android_webview```, there is no code about wide viewport quirk mode, so set ```mWideViewportQuirkEnabled``` to ```false```.